snapcraft/yaml.py

```diff
@@ -152,6 +156,22 @@ def __init__(self):
         if 'architectures' not in self.data:
             self.data['architectures'] = [common.get_arch(), ]
 
+    def _remap_skills(self):
+        if 'uses' in self.data:
+            logger.warning(
+                "DEPRECATED: Instances of 'uses' remapped to 'slots'")
```

Comment:
I'd display a link to the mailing list where Gustavo announces this in case anyone wonders what that is.